tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/AnimalProduct.java

```diff
@@ -0,0 +1,102 @@
+package com.sci.bv.tema14Ianuarie;
+
+
+/**
+ * In this Class are stocked animal products and their characteristics.
+ * (ID, quantity,name of product, list of vitamins, price, validity, weight)
+ */
+
+public class AnimalProduct {
+
+    private int ID;
+    private int Qty;
+    private String Name;
+    private double Temp;
+    private double Price;
+    private int Validity;
```

Comment:
since you're not actually storing a date, why even bother having it as a int? why not just store it as a String?